types/editorjs__list/index.d.ts

```diff
@@ -0,0 +1,76 @@
+// Type definitions for @editorjs/list 1.7
+// Project: https://github.com/editor-js/list#readme
+// Definitions by: Benjamin Wenzel <https://github.com/BenjaminWenzel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.6
```

Comment:
```suggestion
```
it has no impact, at least on within DT ecosystem (too old version, see readme information on this)